### DISCUSSION TITLE
notify.warn [lspconfig] Cannot access configuration for lua_ls

### DISCUSSION THREAD

**Discussion by alezunino** [LazyVim/LazyVim#262] [Q&A] [answered]
Hi,
some hours ago I updated LazyVim (1.9.2) and now when launching nvim (0.8.3) and opening any lua file the following error appears:

`
notify.warn [lspconfig] Cannot access configuration for lua_ls. Ensure this server is listed in `server_configurations.md` or added as a custom server.
`

and the lua lang server does not work. In the git log I found this: https://github.com/LazyVim/LazyVim/commit/543126818e678509ee9e17836f0e7906146a1002 which seems related. I have no other plugins or lsp/mason extra config. Is this a known problem?

thanks

**Comment by folke** [comment, marked as answer]
Make sure to update all your plugins. lspconfig has renamed sumenko_lua to lua_ls.

**Reply by alezunino** [reply]
> Make sure to update all your plugins. lspconfig has renamed sumenko_lua to lua_ls.

Thanks! I was using version = "*" in lazy config. Fixed by using version = false

BTW. LazyVim and lazy.nvim are awesome and well documented!

**Reply by folke** [reply]
right, forgot about that! And thank you! :)
